docs/native/src/scenarios/from_app.rst

```diff
@@ -9,42 +9,45 @@ Opening files from your app in |Office iOS|
 In order to invoke |Office iOS| when opening an Office file from your app, you must use the URL schemes that Word,
 Excel, and PowerPoint for iOS register when they are installed.
 
-General information on the Office URL schemes can be found here:
-https://msdn.microsoft.com/en-us/library/office/dn906146.aspx#sectionSection1
+URL Schemes for Inovking Office
```

Comment:
Typo. If you set up a local build environment you can actually run check_spelling.bat to check your spelling. :smile: